packages/client/components/ScopePhaseArea.tsx

```diff
@@ -121,30 +135,35 @@ const ScopePhaseArea = (props: Props) => {
       >
         <TabContents>
           <ScopePhaseAreaGitHub
-            isActive={activeIdx === 0}
+            isActive={isTabActive('github')}
             gotoParabol={goToParabol}
             meetingRef={meeting}
           />
         </TabContents>
         <TabContents>
           <ScopePhaseAreaJira
-            isActive={activeIdx === 1}
+            isActive={isTabActive('jira')}
```

Comment:
A question, maybe more for Matt (or the Growth team): at the moment, Jira is the default active index. If we want to steer users to Jira, should we make them the first tab on the left, i.e. idx 0 rather than idx 1?

Reply:
I like it! I've created an issue: https://github.com/ParabolInc/parabol/issues/6222